fullstop-plugin-api/pom.xml

```diff
@@ -105,6 +105,11 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
Why another JSON lib? Is jackson-databind not sufficient here? It has much better Spring (Boot) support afaik.